pilot/pkg/model/context.go

```diff
@@ -484,6 +484,9 @@ type NodeMetadata struct {
 	// DNSCapture indicates whether the workload has enabled dns capture
 	DNSCapture string `json:"DNS_CAPTURE,omitempty"`
 
+	// ProxyXDSViaAgent indicates that xds data is being proxied via the agent
+	ProxyXDSViaAgent string `json:"PROXY_XDS_VIA_AGENT,omitempty"`
```

Comment:
We may choose to use this pilot, in future to send something special to the agent knowing that the agent will strip it out before sending to envoy. this is simply an indicator that we are proxying via agent. It will be temporary - once we have this running by default, we can get rid of this value as we will always know that the agent will be receiving the xds.